module1.js

```diff
@@ -53,7 +56,7 @@ function question3(number) {
 */
 
 function question4(array) {
-  // TODO:
+  return array.map(item => item + 1);
```

Comment:
If you look at the expected test result, you'll see that the expected array has fewer elements than the input array - you only want every 2nd element of the input array in the new output array. `.map()` function helps you modify **each** element in the array, and that's you've done here - you incremented each element by 1. So how can you filter out the elements form the array then to make the test pass?